### PR TITLE
Updated the Readme Files

### PR DESCRIPTION
**Pull request by ajlende**
I split up the README.md file into a bunch of smaller README.md files, each specific to its directory.

I also moved the proxy code into it's own directory as well so it's not just floating around out there.

@ajbowler @nmont @ThomasKearney

### CODE REVIEW

File: Java/README.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+We follow the [Google Java Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javaguide.html)
+
+- Brackets follow the Kernighan and Ritchie style with brackets on the same line.
+- `@override` always used.
+- Constant names use `CONSTANT_CASE`.
+- Class names use `UpperCamelCase`.
+- Other variable or method names use `lowerCamelCase`.
+- Javadoc-style comments.
+- Don't be stupid. Others sould be able to read your code.
```
Comment (ajbowler) [inline Java/README.md]: Lol out of all the lines, the don't be stupid line has a typo.
LOL

Comment (ajlende) [inline Java/README.md]: Man. How did I miss that?

**Comment by ajbowler** [comment]
@ajlende Will moving the JS files to a separate Proxy folder break everything due to changing file paths? Just checking since you wrote `server.js`

I realize that it isn't as big a deal at this point in time with no hosting but it's something that would need fixing.

**Comment by ajbowler** [comment]
Also, would it be worth just deleting `client.js` since it's never actually used and it was leftover from testing?

**Comment by ajlende** [comment]
Moving it to the proxy shouldn't make a difference. server.js basically starts up a REST service that gets used internally, so it doesn't matter what directory it's in. I'm leaving client.js in there for now because it was a useful utility for testing, but I'll mention it in the readme.

**Comment by ThomasKearney** [comment]
:+1: 
M(usic)14 breaths in life once again!!
Alt Text

**Comment by ajbowler** [comment]
:+1: 

LOOKS LIKE M14'S BACK ON THE MENU BOYS

**Comment by nmont** [comment]
:+1: 
Oh the nostalgia of Github/Trello notifications. 
After weeks of dormancy -

**Pull request merged**